fastlane/Fastfile

```diff
@@ -217,12 +226,13 @@ REPOSITORY_NAME="WordPress-Android"
   #####################################################################################
   desc "Updates a release branch for a new beta release"
   lane :new_beta_release do | options |
+    app = options[:app]
     android_betabuild_prechecks(options)
     send_strings_for_translation()
-    download_translations(options)
-    android_bump_version_beta()
-    new_version = android_get_app_version()
-    trigger_release_build(branch_to_build: "release/#{new_version}")
+    android_update_metadata(options)
```

Comment:
The revert of `download_translations(options)` line 222 back into `android_update_metadata` is incorrect.
But I have fixed it in [this PR](https://github.com/wordpress-mobile/WordPress-Android/pull/14870/files#diff-dff4b99d4e651ab9d7597876ec8dbe92aa934e42c0fb55f4aadd6c106fc96688) targeting your branch here, so once #14870 is merged this diff should be outdated.